setup.py

```diff
@@ -22,6 +22,7 @@
 
 from setuptools import setup
 from distutils.command.build import build as du_build
+from distutils.command.install import install as du_install
```

Comment:
You may want to use setuptools.command.install here...

Reply:
Right. Thanks.